pipeline/Makefile

```diff
@@ -350,27 +353,27 @@ ecoli_ref.fastq.bz2.keep.assembly.fa: ecoli_ref.fastq.bz2.keep.abundfilt.keep
 	# 	cp ecoli.37/contigs.fa ecoli_ref.fastq.bz2.keep.assembly.fa
 
 orig_vs_ecoli_assembly.report: ecoliMG1655.fa ecoli_ref.fastq.bz2.keep.assembly.fa
-	dnadiff ecoliMG1655.fa ecoli_ref.fastq.bz2.keep.assembly.fa -p orig_vs_ecoli_assembly
+	dnadiff ecoliMG1655.fa ecoli_ref.fastq.bz2.keep.assembly.fa -prefix orig_vs_ecoli_assembly
 
 ecoli_dn.faa: ecoli_ref.fastq.bz2.keep.assembly.fa
-	/mnt/prokka-1.7/bin/prokka ecoli_ref.fastq.bz2.keep.assembly.fa --outdir ecoli_dn --prefix ecoli_dn --force
+	${HOME}/src/prokka/prokka-* ecoli_ref.fastq.bz2.keep.assembly.fa --outdir ecoli_dn --prefix ecoli_dn --force
```

Comment:
@mr-c 
Here where I changed the prokka directories. I thought we talked about installing to `${HOME}/src/prokka/prokka-*/bin`, but both that and `${HOME}/src/prokka/prokka-*` throw the error below. Isn't this bin (next to the binaries) where we want these installs though? I don't think anywhere else is appropriate.

```bash
/home/wrightm/src/prokka/prokka-* sar.keep.assembly.fa --outdir sar_dn --prefix sar_dn --force
/bin/sh: /home/wrightm/src/prokka/prokka-1.11: Is a directory
make: *** [sar_dn.faa] Error 126
```